todo_app/app.py

```diff
@@ -1,11 +1,23 @@
-from flask import Flask
+from flask import Flask , render_template, request
+from flask.helpers import url_for
+from werkzeug.utils import redirect
```

Comment:
I'm slightly surprised to see this - I would have expected to import redirect from flask but it seems to work nonetheless..